kili/services/export/typing.py

```diff
@@ -0,0 +1,27 @@
+"""
+Types used by the conversion service
+"""
+from typing import NamedTuple, NewType, Tuple, Union
+
+from typing_extensions import Literal
+
+LabelFormat = Literal["yolo_v4", "yolo_v5"]
+InputType = Literal["text", "image"]
```

Comment:
We should use this one everywhere. shouldn't we create a typing.py at the root of kili with common types like this one ?
And as the backend interpret it only in upper case, maybe have InputType = Literal["TEXT", "IMAGE", "PDF" , ect...]